pkg/api/cookiestore.go

```diff
@@ -157,3 +158,13 @@ func (cleanTask *CleanTask) DoWork(ctx context.Context) error {
 
 	return nil
 }
+
+func (cleanTask *CleanTask) String() string {
+	return fmt.Sprintf("{SessionCleanup: %s, sessions: %s}",
+		"session clean task", // description of generator's task purpose
+		cleanTask.sessions)
+}
+
+func (cleanTask *CleanTask) Name() string {
```

Comment:
Are the "Name()" methods called somewhere? I looked for the code and could not find the calls.

Reply:
So it is only called by the monitoring logic.
I noticed the "Name()" method in some of the task implementations are not covered.
Wouldn't it be better if "Name()" was also called in all the "String()" implementations?

```
	return fmt.Sprintf("{%s: %s, sessions: %s}",
		cleanTask.Name(),
		"session clean task", // description of generator's task purpose
		cleanTask.sessions)
```

This way we make we'll both increase coverage and have one less place where to hardcode the task/generator name